### ISSUE TITLE
Logo version in mail template

### ISSUE THREAD

**Issue by RandieM** [nextcloud/server#13415]
**Is your feature request related to a problem? Please describe.**
Email templates in NC14 (e.g. reset password email) are using the main logo version for their header.

**Describe the solution you'd like**
Since the header in the email template takes the same colour as the header in the web interface, I think it would make more sense if the header logo was used in email templates too.

**Comment by MorrisJobke** [comment]
cc @nextcloud/designers Does that makes sense? I guess so, as we now have the header logo and the normal logo.

**Comment by jancborchardt** [comment]
Yes, that makes sense, as the email headers are also smaller. If someone can put a pointer to the file where this would need to be changed, this would be a good first issue! :rocket: :slightly_smiling_face:

**Comment by RandieM** [comment]
This is a wild guess (spoiler alert: I don't speak PHP)... but I can see in the file `server/lib/private/Mail/EMailTemplate.php` the code below, which, I believe, fetches the main logo from the themingDefaults (`server/apps/theming/lib/ThemingDefaults.php`). The function getLogo does not seem to accept an argument for which logo type to fetch, nor could I spot another function that fetches the header logo.

```$logoUrl = $this->urlGenerator->getAbsoluteURL($this->themingDefaults->getLogo(false));
$this->htmlBody .= vsprintf($this->header, [$this->themingDefaults->getColorPrimary(), $logoUrl, $this->themingDefaults->getName()]);

**Comment by RandieM** [comment]
It seems to me that this has been fixed in Nextcloud 15.0.4.

**Comment by MorrisJobke** [comment]
> It seems to me that this has been fixed in Nextcloud 15.0.4.

Thanks for the feedback. I will close the ticket for now.
